pyrasterframes/python/examples/NDVI.py

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
As a stopgap for documentation, could you add a page to the user manual (titled, say, "PyRasterFrames"), with links to each of these examples, and a note the more documentation is coming? I'd like to draw attention to its existence and provide support for using with as little effort at this time.